src/res_core.ml

```diff
@@ -2217,59 +2228,84 @@ and parseBinaryExpr ?(context=OrdinaryExpr) ?a p prec =
   (* ) *)
 
 and parseTemplateExpr ?(prefix="js") p =
```

Comment:
It's best to review the changes to this function using the split view and ignore the left side completely as if its contents have been completely replaced.